app/src/main/res/values/strings.xml

```diff
@@ -366,15 +366,15 @@
     <string name="sync_phase_apply_local_entries">reapplying local entries</string>
     <string name="sync_phase_push_entries">pushing entries</string>
     <string name="sync_phase_post_processing">post processing</string>
-    <string name="sync_error_unauthorized">Authentication failed</string>
-    <string name="sync_error_user_inactive">User is inactive</string>
+    <string name="sync_error_unauthorized">Could not log in</string>
```

Comment:
```suggestion
    <string name="sync_error_unauthorized">Could not confirm identity</string>
```

Reply:
Still not a fan. :|

Reply:
If you explain it above I can add the translator info and come up with something that encompasses all uses of "authenticate". "Authenticate" and "Unencrypt" with different passwords arrives at failure in comprehension for no good reason.

Reply:
Let's keep the discussion in the other thread (sorry for replying here too, I thought that one was gone). I'll reply there now.